### DISCUSSION TITLE
Steps to Troubleshoot Linux Client?

### DISCUSSION THREAD

**Discussion by kendoori** [jamulussoftware/jamulus#2648] [Technical Support & Help] [answered]
I have not used Jamulus in a while, as I've been using Sonobus, but I would like to try again assuming I will not host my own server. I have used the KX repos and have QJackctrl, and everything is properly set up for Sonobus with an M-Audio 192/14. When I open the Jamulus client 3.8.2 I don't see any input activity on the meters, whether I'm connected to a server or not. How can I troubleshoot this?

This may help:

![image](https://user-images.githubusercontent.com/2447420/173874673-ca87eb21-d114-48bf-8ea9-0e60e83a50c1.png)

**Comment by rdica** [comment]
Are you using the same `system:capture_1` and `system:capture_2` ports as inputs for Sonobus as well?

Can you post a screenshot of the Connections window in qjackctl when sonobus is connected?

**Comment by kendoori** [comment]
![image](https://user-images.githubusercontent.com/2447420/173900774-93db5dcf-4aeb-42cf-bcb2-c130ee2b8d06.png)

**Reply by rdica** [reply]
Sorry, this doesn't show me what I expected to see.

Are you sure `system:capture_1` and `system:capture_2` are the correct ports to connect to Jamulus inputs?
You'd only hear something if you were connected to a server in the first configuration.

**Reply by pljones** [reply]
> You'd only hear something if you were connected to a ser

And note, that's the _correct_ set up.  If you can hear your local sound that you want transmitted when not connected, you're set up is wrong.

**Reply by kendoori** [reply]
So I connected to a public server that had at least one active user on it, and ran QjackCtrl on my side and when in the session the input meters showed nothing. I subsequently tore down Jamulus and did the same with Sonobus and I could see my input. I did have this working previously with a different and much simpler interface and I suspect that there is something about how the M-Audio is interacting with Jamulus via Jack...

**Reply by kendoori** [reply]
I figured it out. When I'd used Jamulus before it was through a 2 channel interface. My new interface has 6+ channels. The instruments that I was testing with were not attached to the first two default channels. When I manually patched the other channels to Jamulus using the Carla patchbay function it worked.
![image](https://user-images.githubusercontent.com/2447420/174819529-82e5bb59-2a22-4ece-89e3-d391ef8288af.png)

**Reply by ann0see** [reply]
@kendoori 
Could you please comment with your solution and mark it as answer. Afterwards you can remove the [Solved] tag. 

This makes finding the solution easier

**Comment by kendoori** [comment, marked as answer]
Using Carla patchbay I was able to determine that all the capture inputs were not going to the Jamulus client. I dragged those inputs to Jamulus and it works properly now.
